modules/create-lokistack-cr-cli.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-lokistack-cr-cli_{context}"]
+= Creating a LokiStack custom resource by using the CLI
+
+You can create a `LokiStack` custom resource (CR) by using the {oc-first}.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {loki-op}.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a `LokiStack` CR:
++
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  size: 1x.small # <1>
```

Comment:
This size should be '1x.extra-small' based on the component replicas listed below for Loki.

Reply:
This change is not needed if the below suggestion is incorporated :)